PriorityPro/Todo-App-PriorityPro-Web
# Added motion when adding an item | Fixed some bugs within Cypress

## Overview

**Issue Type**


**Description**
Used the library React Spring to add some animation when adding todos to the list. Fixed our test within cypress to all be passing as before since we added the navbar it was checking and clicking on the `<li>` elements in the navbar when we were running test to click on the `<li>` elements within the list.



**Steps to  Validate Feature **

1. Add a todo item
2. Notice the animation that is being created upon adding an item
3. 

**Previous behavior**
When adding an item there was no animation.

**Expected behavior**
Adding, editing and deleting an item should work as expected.

**Screenshots & Videos**

https://github.com/PriorityPro/Todo-App-PriorityPro-Web/assets/90209258/3de9976d-0812-475e-98af-8782df78fad1
